stable/weave-scope/values.yaml

```diff
@@ -0,0 +1,76 @@
+  imageTag: "1.6.5"
+# global.service.*: the configuration of the service used to access the frontend
+  service:
+# global.service.name: (required if frontend.enabled == true) the short name desired for the frontend service
+# global.service.name is a global so we can access its value easily from the agent subchart
+    name: "weave-scope-app"
+# global.service.port: (required if frontend.enabled == true) the port exposed by the Scope frontend service
+# global.service.port is a global so we can access its value easily from the agent subchart
+    port: 80
+# global.service.type: (required if frontend.enabled == true) the type of the frontend service -- must be ClusterIP, NodePort or LoadBalancer
+# global.service.type is a global to keep it with the other values for configuring the frontend service
+    type: "ClusterIP"
+#
+# weave-scope-frontend.* controls how the Scope frontend is installed
+weave-scope-frontend:
+  enabled: true
+# weave-scope-frontend.imagePullPolicy: controls when the image is pulled
+  imagePullPolicy: ""
+# weave-scope-frontend.containerPort: (required if frontend.enabled == true) the port opened to the Scope frontend
+  containerPort: 4040
+# weave-scope-frontend.resources.*: controls requests/limits for the frontend
+# weave-scope-frontend.resources.* values are all optional but should not be set to empty values
+#  resources:
+#    requests:
+# weave-scope-frontend.resources.requests.cpu: CPU req. in MHz (m)
+#      cpu: ""
+# weave-scope-frontend.resources.requests.memory: memory req. in MiB (Mi)
+#      memory: ""
+#    limits:
+# weave-scope-frontend.resources.limits.cpu: CPU limit in MHz (m)
+#      cpu: ""
+# weave-scope-frontend.resources.limits.memory: memory limit in MiB (Mi)
+#      memory: ""
+#
+# weave-scope-agent.* controls how the Weave Scope node agent pods are installed
+weave-scope-agent:
+  enabled: true
+# weave-scope-agent.imagePullPolicy: controls when the image is pulled
+  imagePullPolicy: ""
+# weave-scope-agent.dockerBridge: (required if agent.enabled == true) the name of the Docker bridge interface
+  dockerBridge: "docker0"
+# weave-scope-agent.scopeFrontendAddr: the host:port of a Scope frontend to send data to
+# weave-scope-agent.scopeFrontendAddr is only needed for some cases where the frontend is deployed separately from the agent
+  scopeFrontendAddr: ""
+# weave-scope-agent.probeToken: the token used to connect to Weave Cloud
+# weave-scope-agent.probeToken is not needed for connecting to non-cloud Scope frontends
+  probeToken: ""
+# weave-scope-agent.rbac.*: controls RBAC resource creation/use
+  rbac:
+# weave-scope-agent.rbac.create: whether RBAC resources should be created
+# weave-scope-agent.rbac.create *must* be set to false if RBAC is not enabled in the cluster
+# weave-scope-agent.rbac.create *may* be set to false in an RBAC-enabled cluster to allow for external management of RBAC
+    create: true
```

Comment:
Default should be `false`.

Reply:
RBAC is enabled by kubeadm by default starting with 1.6 (other cluster installers like Tectonic enabled it earlier than that, even) so I figured with that change the default for rbac.create should match the default for a cluster -- but if you feel strongly about it, I'll change it.  (One of the example chart PRs you linked to above has it false, one has it true, so I don't know if there's a true convention around it yet.)

Reply:
Hm, I checked several charts and only found it to be `false` by default. I don't have a strong opinion on this.

Reply:
I filed kubernetes/helm#3098 on this.